### DISCUSSION TITLE
Celery Worker exited with 'signal 9 '

### DISCUSSION THREAD

**Discussion by Tangruilin** [apache/airflow#33106] [General]
### Apache Airflow version

2.6.3

### What happened

I deploy airflow on one machine and celery on another machine. When i start a example, it raise error like the pic

![L1VzZXJzL3RhbmdydWlsaW4vTGlicmFyeS9BcHBsaWNhdGlvbiBTdXBwb3J0L2lEaW5nVGFsay8zODAwMTk0NDkzX3YyL0ltYWdlRmlsZXMvMTY5MTEzNzA0MTMwNl81RTE2MzFENS1FRkFBLTQyRTUtQTMyNC1FQzkxNjY1QjQxRjkucG5n](https://github.com/apache/airflow/assets/23651891/76ea9be7-a583-4576-a523-ea04d4f3322d)

### What you think should happen instead

the task should run nornamally

### How to reproduce

I don't know, for it work nornamally on another machine

### Operating System

Centos7

### Versions of Apache Airflow Providers

2.6.3

### Deployment

Official Apache Airflow Helm Chart

### Deployment details

I install it with python 3.9.9 and  python 3.7.2, but they both error

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Reply by potiuk** [reply]
Converting to a discusison. Pleaase do not open issues when you want to get help in troubleshooting your problem (as clearly advised when you open the issue) - next time open a discussion if you are not sure if this is an airflow issue or troubleshooting your installation.

Be mindful of people who try to help others here in their free time. Opening issues when the template clearly expects you to provide reproducible case and if you are not able - directs you to open discussion instead, is the least you can do to make it easier for those who try to help others in their free time.

Back to your issue - something killed your worker with sigkill. Likely this is your deployment (whatever it is) decided to kill it - either because of hardware or resource problem. Look at your K8S logs to determine the cause. There is nothing Airflow or Helm can do about it (or even know about it) - you as deployment manager need to monitor and manage your deployment to determine root causes for any events of that kind and adjust your resources and K8S configuration accordingly.

**Reply by potiuk** [reply]
Also see https://airflow.apache.org/docs/apache-airflow/stable/installation/index.html#using-official-airflow-helm-chart which explains what is expected of someone who uses particular deployment methods of Airflow.

> You need to setup monitoring of your system allowing you to observe resources and react to problems.

> You are expected to configure and manage appropriate resources for the installation (memory, CPU, etc) based on the monitoring of your installation and feedback loop.
